js/id/renderer/background_source.js

```diff
@@ -117,7 +117,7 @@ iD.BackgroundSource.Bing = function(data, dispatch) {
     };
 
     bing.logo = 'bing_maps.png';
-    bing.terms_url = 'http://opengeodata.org/microsoft-imagery-details';
+    bing.terms_url = '//opengeodata.org/microsoft-imagery-details';
```

Comment:
Looks like this has moved to:
https://blog.openstreetmap.org/2010/11/30/microsoft-imagery-details/